uber/site_sections/registration.py

```diff
@@ -658,12 +658,12 @@ def shifts(self, session, id, shift_id='', message=''):
             'message':  message,
             'shift_id': shift_id,
             'attendee': attendee,
-            'possible': attendee.possible_opts,
             'shifts':   Shift.dump(attendee.shifts),
             'jobs':     [(job.id, '({}) [{}] {}'.format(custom_tags.timespan.pretty(job), job.location_label, job.name))
                          for job in session.query(Job)
                                            .outerjoin(Job.shifts)
                                            .filter(Job.start_time > localized_now() - timedelta(hours=2),
```

Comment:
Brent asked for this page to only show "relevant" shifts, meaning shifts from the same department.